### DISCUSSION TITLE
Is it possible to add in the devcontainer name into the build directory automatically?

### DISCUSSION THREAD

**Discussion by ferenj-gs** [microsoft/vscode-cmake-tools#3715] [Q&A] [answered]
Hey,

We have multiple devcontainers (using the Devcontainers extension) that targets different build targets and architectures. Whenever we switch container  we need to clean cache and recocfigure the projects as the build/ directory is common.
Is it possible to setup the cmake-tools extension to add in the active devcontainer name as a suffix to the build directory so when we switch devcontainers we can proceed with zero overhead ? 
E.g two containers named : `x86-64` and `arm64`  would use `${workspaceFolder}/build-x86-64` and `${workspaceFolder}/build-arm64` respectively?

Thanks & BR,

Ferenc

**Comment by gcampbell-msft** [comment, marked as answer]
@ferenj-gs There is not any support for this from the extension currently. 

From a very quick search, I don't see a way to interact with the Devcontainers extension in that way, and so I'm not sure this will be able to be supported.
